iris/service/content/file/document.rst

```diff
@@ -0,0 +1,27 @@
+===================
+File Document Tests
+===================
+
+File objects are not real file object, but objects containing meta data for
+files stored on an external location (e.g. S3).
+
+Create a file object with valid data::
+
+    >>> from iris.service.content.file.document import File, StorageType
+    >>> f = File(
+    ...     id="some_hash",
+    ...     state='visible',
+    ...     original_name="sample.txt",
+    ...     owner_id="some_owner",
```

Comment:
owner_id is an internal property, the owner must be set using the owner property.
Maybe it would be better to change the property name to '_owner_id'.